README.md

```diff
@@ -13,8 +13,37 @@ Python wrapper for vxl using pybind11
 pyvxl can be build standalone or as a module of an existing project.
 
 ## Standalone
+
+Whatever environment (your host or a docker container) you build pyvxl in will require these dependencies:
+- g++
+- Python 3 development libraries (e.g. `python3-devel`)
+- GeoTIFF development libraries (e.g. `libgeotiff-devel`)
+
+If you're on your host, it's recommended you do this in a virtualenv so you don't mess with your
```

Comment:
missing the end of the sentence?

Reply:
Thanks that sentence was replaced by the virtualenv section below.